src/components/ViewerMessaging/MessagingDefinitionsSummary.tsx

```diff
@@ -27,7 +27,24 @@ export default function MessagingDefinitionsSummary(
     return (
         <div>
             {Object.keys(filteredDefinitions)
-                .sort((a, b) => a.localeCompare(b))
+                // Rudimentary sort of components / services first and then the
+                // properties should all come after TODO: Better sort so
+                // components are in one section and services are in another
+                .sort((a, b) => {
+                    if (
+                        a.toLocaleLowerCase().includes("modelproperties") &&
```

Comment:
For performance we could do this once at the start
```
const aHasModelProps = a.toLocaleLowerCase().includes("modelproperties");
const bHasModelProps = b.toLocaleLowerCase().includes("modelproperties");
```

Reply:
Fixed.